### PR TITLE
[58] Develop an E-commerce Store Database

### PR DESCRIPTION
**Pull request by Israel-Force**
## Task: `Database Task`
- `added .sql file`

### CODE REVIEW

File: submissions/israelOladosu/database/init.sql [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    order_id INT NOT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(product_id) REFERENCES products(id) ON DELETE CASCADE,
+    FOREIGN KEY(order_id) REFERENCES orders(id) ON DELETE CASCADE
+);
+
+----------------------------------------------------INSERT INTO TABLES-------------------------------------------------------
+
+-- insert into categories table 
+INSERT INTO categories (category_name)
```
Comment (TundeOladejo) [inline submissions/israelOladosu/database/init.sql:89]: The table name here doesn't match the table you created. Check line 6.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/israelOladosu/database/init.sql (hunk 1 above)
Comment (Israel-Force) [reply submissions/israelOladosu/database/init.sql:89]: Thank you... I have made the needed changes.

File: submissions/israelOladosu/database/init.sql [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    phone_number VARCHAR(21) NOT NULL,
+    customer_id INT,
+    PRIMARY KEY(id),
+    FOREIGN KEY(customer_id) REFERENCES customers(id) ON DELETE CASCADE
+);
+
+-- orders table
+CREATE TABLE orders (
+    id INT NOT NULL AUTO_INCREMENT,
+    order_amount INT NOT NULL,
```
Comment (adaugochi) [inline submissions/israelOladosu/database/init.sql:66]: well done, please use `DECIMAL` for amount. Find out how it is used

Comment (Israel-Force) [reply submissions/israelOladosu/database/init.sql]: Done... Thank you

File: submissions/israelOladosu/database/init.sql [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+INSERT INTO categories (category_name)
+VALUES('farm animals'),
+      ('crops'),
+      ('cereals'),
+      ('vegetables'),
+      ('tubers');
+
+-- insert into admins
+INSERT INTO admins (firstname, lastname, email, admin_password, phone_number)
+VALUES ('Nifemi', 'Omotola', 'nifemi@email.com', 'omotola', 08012345678),
```
Comment (adaugochi) [inline submissions/israelOladosu/database/init.sql:98]: phone_number is a `varchar`. kindly wrap the phone number in quotes

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**